tests/security/eal4/syscall_thrasher.pm

```diff
@@ -12,12 +12,20 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use Utils::Architectures 'is_s390x';
 
 my $log_file = '/tmp/syscalls_output.log';
 
 sub run {
     my ($self) = shift;
 
+    # Skip on s390x with encrypted disk due to timeouts
+    # poo#176454
+    if (get_var('ENCRYPT') && is_s390x) {
```

Comment:
one last minor bit: I'd be more precise and check for the value like `check_var('ENCRYPT', '1')` .